src/shell/shell.cpp

```diff
@@ -1325,11 +1327,11 @@ void SHELL_Init() {
 		"  [color=light-green]for[reset] [color=white]%VAR[reset] [color=light-cyan]in[reset] [color=white](SET)[reset] [color=light-cyan]do[reset] [color=white]COMMAND[reset]\n"
 		"\n"
 		"Where:\n"
-		"  [color=white]%VAR[reset]    A single character representing a variable, prefixed by a '%'.\n"
-		"  [color=light-cyan]in[reset]      A case-insensitive keyword.\n"
-		"  [color=white](SET)[reset]   The set of strings to replace [color=white]%VAR[reset] instances in [color=white]COMMAND[reset].\n"
-		"  [color=light-cyan]do[reset]      A case-insensitive keyword.\n"
-		"  [color=white]COMMAND[reset] The command to be repeated for each string in [color=white](SET)[reset].\n"
+		"  [color=white]%VAR[reset]    is a single character representing a variable, prefixed by a '%'.\n"
+		"  [color=light-cyan]in[reset]      is a case-insensitive keyword.\n"
+		"  [color=white](SET)[reset]   is the set of strings to replace [color=white]%VAR[reset] instances in [color=white]COMMAND[reset].\n"
+		"  [color=light-cyan]do[reset]      is a case-insensitive keyword.\n"
+		"  [color=white]COMMAND[reset] is the command to be repeated for each string in [color=white](SET)[reset].\n"
```

Comment:
Remove all the redundant "is" words, remove periods, as per above. Just use sentence fragments, which is perfectly fine in such cases (also for bulleted lists, that's perfectly legal in English).